### PR TITLE
Feature/hide users with 0 hours

### LINKED ISSUE #30: Hide users who have logged 0 hours

### PR DESCRIPTION
**Pull request by schmierkov**
@mauro-oto @etagwerker this PR fixes #30, check it out! Thanks

### CODE REVIEW

File: app/controllers/leaderboard_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class LeaderboardController < ApplicationController
   def users
-    @leaderboards = UserLeaderboard.includes(:user).week(start_date, end_date)
+    @leaderboards = UserLeaderboard.includes(:user).week(start_date, end_date).
+      with_logged_minutes
```
Comment (etagwerker) [inline app/controllers/leaderboard_controller.rb]: @schmierkov We usually use Option A for multi-line method chaining: https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains

**Comment by etagwerker** [comment]
@schmierkov Other than that one comment it looks good!

**Comment by schmierkov** [comment]
@etagwerker thanks for the feedback, I always get confused which version we prefer. just updated the code, thanks! 👍

**Comment by etagwerker** [comment]
@schmierkov Great, thanks! 🍻

**Comment by etagwerker** [comment]
@schmierkov Could you solve #32 too? It'll be a very similar solution.

**Comment by schmierkov** [comment]
@etagwerker I didn't see that this exists for projects too! I will check asap, thanks for pointing it out!

**Pull request merged**